contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -76,14 +72,19 @@ class ExpectColumnValuesToBePresentInAnotherTable(QueryExpectation):
     }
 
     metric_dependencies = ("query.template_values",)
-    template_dict: dict
+
+    foreign_key_column: str
+    foreign_table: str
+    primary_key_column_in_foreign_table: str
+
+    template_dict: dict = {}
+
     query = """
-        SELECT COUNT(1) FROM (
```

Comment:
Interesting enough,  this was the only change that was needed to retrieve the full set of unexpected values